nvdaHelper/vbufBase/backend.cpp

```diff
@@ -272,6 +276,18 @@ VBufStorage_controlFieldNode_t* VBufBackend_t::reuseExistingNodeInRender(VBufSto
 		LOG_DEBUG(L"Existing node refuses to be reused");
 		return nullptr;
 	}
+	// We only allow reusing nodes that remain at the same position in the tree.
+	// I.e. We never reuse a node that is from  a different parent.

```

Comment:
Even though you mention parent here, the term "position" is used, which might make someone think this also applies to movement within the same parent. You could either just shorten this to talk about the same parent or clarify by saying that movement within the same direct parent is still okay.